stable_baselines3/common/policies.py

```diff
@@ -350,7 +350,7 @@ def predict(
 
         # Remove batch dimension if needed
         if not vectorized_env:
-            actions = actions[0]
+            actions = actions.squeeze(0)
```

Comment:
```suggestion
            actions = actions.squeeze(axis=0)
```

Please use explicit keyword when using numpy ;) unless you have a good reason not to?

Reply:
To be fair, I wanted to ([5009949](https://github.com/DLR-RM/stable-baselines3/pull/964/commits/50099496ee3df53f0f9023ff7b700362dbe58f5b)), but I saw this

https://github.com/DLR-RM/stable-baselines3/blob/a6f5049a99a4c21a6f0bcce458ca3306cef310e0/stable_baselines3/common/distributions.py#L583

Is there a reason to you explicit keyword using numpy and not torch?